packages/wordcount/src/index.ts

```diff
@@ -103,21 +102,25 @@ function countCharacters( text, regex, settings ) {
 /**
  * Count some words.
  *
- * @param {string}                  text         The text being processed
- * @param {WPWordCountStrategy}     type         The type of count. Accepts 'words', 'characters_excluding_spaces', or 'characters_including_spaces'.
- * @param {WPWordCountUserSettings} userSettings Custom settings object.
+ * @param text         The text being processed
+ * @param type         The type of count. Accepts 'words', 'characters_excluding_spaces', or 'characters_including_spaces'.
+ * @param userSettings Custom settings object.
  *
  * @example
- * ```js
+ * ```ts
  * import { count } from '@wordpress/wordcount';
  * const numberOfWords = count( 'Words to count', 'words', {} )
  * ```
  *
- * @return {number} The word or character count.
+ * @return The word or character count.
  */
-export function count( text, type, userSettings ) {
+export function count(
+	text: string,
+	type: Strategy,
+	userSettings: UserSettings
```

Comment:
This `userSettings` should be optional here as it historically has been. Let us create another PR to fix it.

```ts
userSettings?: UserSettings
```

Reply:
Ok sure